server/supabase/migrations/20240608204813_helper_party.sql

```diff
@@ -0,0 +1,76 @@
+create table
+helper_parties (
+uuid uuid default gen_random_uuid() primary key,
+display_name varchar(255) unique not null,
+created_at timestamp default current_timestamp not null
+);
+
+alter table helper_parties enable row level security;
+
+create policy "Helper Parties are visible to authenticated users"
+on helper_parties for select
+to authenticated
+using ( true );
+
+create policy "Helper Parties are only created by authenticated users"
+on helper_parties for insert
+to authenticated
+with check ( true );
+
+create policy "Helper Parties are only updated by authenticated users"
+on helper_parties for update
+to authenticated
+using ( true )
+with check ( true );
+
+create table
+helper_party_networks (
+uuid uuid default gen_random_uuid() primary key,
+display_name varchar(255) unique not null,
+size smallint not null,
```

Comment:
It feels to me that an enum will be more appropriate here - you may want to support semi-honest/malicious 2 party, TEE, etc.. Size will require a migration in the future

Reply:
yea, that makes sense. at this point, I don't think this is used so maybe it makes sense to just drop it until we need it.